src/components/app/ListOfPublishedProfiles.js

```diff
@@ -254,12 +257,17 @@ export class ListOfPublishedProfiles extends PureComponent<Props, State> {
         <>See and manage all your recordings ({profilesRestCount} more)</>
       );
     } else {
-      profileRestLabel =
-        uploadedProfileInformationList.length > 1 ? (
-          <>Manage these recordings</>
-        ) : (
-          <>Manage this recording</>
-        );
+      profileRestLabel = (
+        <Localized
+          id="ListOfPublishedProfiles--uploaded-profile-information-label"
+          vars={{
+            uploadedProfileInformationListLength:
+              uploadedProfileInformationList.length,
+          }}
+        >
+          {'{ $ uploadedProfileInformationListLength } Manage these recordings'}
```

Comment:
As we discussed earlier, this should be `$uploadedProfileInformationListLength`. See the space after the dollar sign.